tests/utils_tests/test_restart_with_reloader.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
Please put this test in the existing `test_autoreload.py` file in the same directory.